SPECS/mercurial/mercurial.spec

```diff
@@ -51,14 +51,21 @@ popd
 %postun
 /sbin/ldconfig
 
+%define zsh_completions_dir %(pkg-config --variable=completionsdir zsh 2>/dev/null || echo '%{_datadir}/zsh/site-functions')
```

Comment:
nit: adding comments for why this / other changes were included would be helpful